projects/core/src/lib/mask.ts

```diff
@@ -23,15 +23,9 @@ export class Maskito extends MaskHistory {
         ...this.maskitoOptions,
     };
 
-    private readonly preprocessor = maskitoPipe(
-        this.options.preprocessor,
-        ...this.options.preprocessors,
-    );
+    private readonly preprocessor = maskitoPipe(...this.options.preprocessors);
```

Comment:
Why spread?

Reply:
Should we change signature of `maskitoPipe` ? 🤔 

```ts
export function maskitoPipe(
    processors: ReadonlyArray<MaskitoPreprocessor | null | undefined>
): MaskitoPreprocessor;

export function maskitoPipe(
    processors: ReadonlyArray<MaskitoPostprocessor | null | undefined>
): MaskitoPostprocessor;
```

What do you think ?

Reply:
Done in https://github.com/Tinkoff/maskito/pull/337/commits/593a9313e56521383520e0db2c07023fe55d1449

Could you review the PR again, please ?